core/src/Threads/Kokkos_Threads_ParallelScan_Range.hpp

```diff
@@ -21,6 +21,34 @@
 
 namespace Kokkos {
 namespace Impl {
+namespace Details {
```

Comment:
Is it necessary to introduce this namespace? Is `Impl` namespace not enough?

Reply:
Yeah we don't really need more or?